projects/packages/forms/src/contact-form/class-util.php

```diff
@@ -276,6 +278,25 @@ public static function jetpack_tracks_record_grunion_pre_message_sent( $post_id
 
 		/** This action is documented in jetpack/modules/widgets/social-media-icons.php */
 		do_action( 'jetpack_bump_stats_extras', 'jetpack_forms_message_sent', $extra );
+
+		$form_type = 'post';
```

Comment:
What does `post` type refer to? Unknown before type was added, shortcode or something else? Might help to add code comment above.

Reply:
I expect it to track this: 

`post` - form added directly to a post/page content
`widget` - form added via widget
`block_template` - form added to a site-wide template (like Single or 404 templates)
`block_template_part` - form added to a reusable template part

Reply:
It wouldn't let us compare easily widget vs block usage tho in a single graph? We'd need to do math to combine the two latter trend lines.

So it would make sense to have one for block+widget and another for template type?